src/main/docs/guide/gettingStarted.adoc

```diff
@@ -1,10 +1,14 @@
 To enable this support you should add the following dependencies to your build configuration:
 
 dependency:micronaut-openapi[scope="annotationProcessor", version="{version}", groupId="io.micronaut.configuration"]
+for Groovy:
+dependency:micronaut-openapi[scope="compile", version="{version}", groupId="io.micronaut.configuration"]
 
 Then the following compile time dependency:
 
 dependency:swagger-annotations[scope="implementation", groupId="io.swagger.core.v3"]
+for Groovy:
+dependency:swagger-annotations[scope="compile", groupId="io.swagger.core.v3"]
```

Comment:
Please use `implementation` instead of `compile`

Reply:
Doesn't work in my case. Won't generate views

Reply:
Since Gradle 5 (I think) the recommendation is to use `implementation`, not `compile`. They are pretty much the same. Please google for differences about that if you are interested in learn more.

Reply:
> Since Gradle 5 (I think) the recommendation is to use `implementation`, not `compile`. They are pretty much the same. Please google for differences about that if you are interested in learn more.

I will retry this but as stated it does not generate views if using implementation.

Reply:
Please see my other comment. It works.

Reply:
Apparently I left **annotationProcessor** for open API and that's the reason I didn't work, not the implementation on swagger. Sorry about that.